src/views/About.vue

```diff
@@ -627,6 +557,126 @@ ent-residents/"
 export default {
   components: {
     // HowItWorks
+  },
+  data() {
+    return {
+      publicHealth: [
```

Comment:
We can eventually store this data in firebase and then pull down and populate. But I was going to save that for a different issue.